app/containers/MessageBox/UploadModal.js

```diff
@@ -119,6 +119,10 @@ export default class UploadModal extends Component {
 	submit = () => {
 		const { file, submit } = this.props;
 		const { name, description } = this.state;
+		if (name === '') {
+			ToastAndroid.show('Please enter a filename', ToastAndroid.SHORT);
```

Comment:
What about iOS?

Reply:
@diegolmello i have changes Toast to alert.